.env.example

```diff
@@ -0,0 +1,28 @@
+# You'll need to set the following in either your environment or in a .env
+# file in the root of the project
+#
+# The values in this file are purely examples to better help understand what it
+# is you need to add
+
+PORT=8000
```

Comment:
I know its obvious now, but for the possible future me that has to get this thing running with no one to hold my hand 😟 , could we add a brief comment as to what port we are referring to.

Also should we be more explicit in the name e.g. WP_PORT or WASTE_PERMITS_PORT or THE_WEB_APPLICATION_SERVER_PORT_FOR_THE_WASTE_PERMITS_DIGITAL_SERVICE. Open to suggestions (though don't think you'll beat my last one!)